Makefile

```
@@ -8,11 +8,27 @@ ifeq ($(AUTH_FILE),)
 AUTH_FILE = /etc/serviceaccount/service-account.json
 endif
 
+ifdef PULL_PULL_SHA
+GITHUB_SHA = $(PULL_PULL_SHA)
+endif
+
 export GOOGLE_APPLICATION_CREDENTIALS=$(AUTH_FILE)
 
 .PHONY: deploy clean
 deploy: nodepool_create resource_apply
-clean: resource_delete nodepool_delete
+clean: nodepool_delete resource_delete
```

Did you check if we can have it like this?

```suggestion
clean: nodepool_delete
```

---

`resource_delete` is deleting the namespace(`prombench-<PR_NUM>`) and the clusterrole, these are not part of the nodepool. so we need to run `resource_delete` anyway. is that right?

---

yep, I think that was it. I remembered now, maybe add a comment somewhere.